packages/truffle/webpack.config.js

```diff
@@ -40,26 +40,34 @@ module.exports = {
       "@truffle/core",
       "index.js"
     ),
+    consoleChild: path.join(
```

Comment:
Just out of curiosity, why do we need to create a bundle for this? It it because Truffle needs to use an external bundle to run the command in a separate process? It seems like this will drastically increase the size of the stuff we need to include with Truffle, huh?

Reply:
I'm understanding that the bundle will be running this other bundle using the user's input. Yeah? This will make the distributed files a lot larger, right? I'm not trying to hold this PR up, I'm just curious.

Reply:
I'm not sure I follow. The file being bundled is `console-child.js`, which is fairly short. The `cli.bundled.js` file is a tiny bit bigger -- 22.842243 megabytes on `develop` and  22.843334 megabytes on this branch. I don't think that's particularly significant?

More to the point though, how else would you get the correct path here?

Reply:
Yes, but `console-child.js` is just the entry point. So it will double the size of the distributed stuff for Truffle. Essentially this will be like a copy of Truffle's main `cli.bundled.js`...which might be ok. I just wanted to point out that this seems to be a significant increase in size. Does that make sense?

Reply:
Well I checked and it is not quite double, it increases the stuff in the build folder from ~50MB to like ~72MB. Something like that.

Reply:
Hm, yeah I think that makes sense but I don't think there's another way. The whole point is that we can run Truffle commands through the child process. So we need all of that bundled this way. If you have a suggestion for how to reduce the distributed code size I'm happy to discuss further!

Reply:
No it's probably fine. I just wanted to bring it up :)

Reply:
Yeah, let's avoid having that bundle in two places :)